Shared/Services/LoopStatsService.cs

```diff
@@ -0,0 +1,44 @@
+﻿using Lexplorer.Models;
+using Newtonsoft.Json;
+using Microsoft.Extensions.Configuration;
+using RestSharp;
+using System.Diagnostics;
+using System;
+using System.Threading;
+using System.Net.Http;
+using System.Net.Http.Json;
+using System.Threading.Tasks;
+using System.Net;
+using Lexplorer.Exceptions;
+
+namespace Lexplorer.Services;
+
+public class LoopStatsService : ILoopStatsService
```

Comment:
Can you implement IDisposable like the other services that use RestSharp?

Reply:
Will do :)

Reply:
Implemented 👍🏻